### DISCUSSION TITLE
Forge mod ID not matching requested ID

### DISCUSSION THREAD

**Discussion by WeebsGames** [Sinytra/Connector#387] [Q&A] [answered]
The requested ID is 47.1.3, however curseforge only allows me to set the version of forge to 47.1.30 which doesn't match. Is there any way to fix this? I am trying to make a custom forge modpack and saw a fabric movement mod that I would like to add to it. There was someone else asking the same question but they haven't gotten a reply in 3 weeks. 
![image](https://github.com/Sinytra/Connector/assets/68392559/0e0101be-28c7-452f-b927-bf21d42fac21)

**Comment by RoomAlex68** [comment, marked as answer]
Do you have Forge 47.1.3 installed??

**Reply by WeebsGames** [reply]
I have forge 47.1.30 installed

**Reply by WeebsGames** [reply]
wait im such an idiot i was thinking decimals not version numbers i just had to look at older versions
